guides/source/getting_started.md

```diff
@@ -367,50 +373,129 @@ If we go back to our browser and make a request to
 
 ### Setting the Application Home Page
 
-Now that we have made the controller and view, we need to tell Rails when we
-want "Hello, Rails!" to show up. In our case, we want it to show up when we
-navigate to the root URL of our site, <http://localhost:3000>. At the moment,
-"Yay! You're on Rails!" is occupying that spot.
+Now that we have made the route, controller, action, and view, let's make a
+small change to our routes. In this application, we're going to change it so
+that our message appears at <http://localhost:3000/> and not just
+<http://localhost:3000/articles>. At the moment, at <http://localhost:3000> it
+still says "Yay! You're on Rails!".
 
-Next, you have to tell Rails where your actual home page is located.
+To change this, we need to tell our routes file where the _root path_ of our
+application is. If we do not do this, Rails will continue to display the "Yay!
+You're on Rails!" page.
 
 Open the file `config/routes.rb` in your editor.
 
 ```ruby
 Rails.application.routes.draw do
-  get 'welcome/index'
-
+  get "/articles", to: "articles#index"
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
 end
 ```
 
-This is your application's _routing file_ which holds entries in a special
-[DSL (domain-specific language)](https://en.wikipedia.org/wiki/Domain-specific_language)
-that tells Rails how to connect incoming requests to
-controllers and actions.
-Edit this file by adding the line of code `root 'welcome#index'`.
-It should look something like the following:
+Underneath the line that is there already, add this line:
 
 ```ruby
 Rails.application.routes.draw do
-  get 'welcome/index'
+  get "/articles", to: "articles#index"
 
-  root 'welcome#index'
+  root to: "articles#index"
+end
+```
+
+A slightly shorter way of writing the same thing is:
+
+```ruby
+Rails.application.routes.draw do
+  get "/articles", to: "articles#index"
+  root "articles#index"
```

Comment:
Maybe we should only show the shorter version of the route. The difference is very subtle and it can only confuse the reader.

Reply:
Good point. The current guide also does not mention the longer version at all. I will remove the longer one, thanks!

Reply:
Fixed.